common/membership/interfaces.go

```diff
@@ -103,6 +103,8 @@ type (
 		RemoveListener(name string) error
 		// MemberCount returns the number of known hosts running this service.
 		MemberCount() int
+		// AvailableMemberCount returns the number of hosts running this service that are accepting requests (not draining).
+		AvailableMemberCount() int
```

Comment:
Can you see if there are other places that uses the MemberCount(). 
For example cluster_aware_quota_calculator.go and

Reply:
I checked for all calls to that method on that interface, but I see the problem now, it's going through a sub-interface. I'll fix this